python/ray/air/util/check_ingest.py

```diff
@@ -93,6 +93,9 @@ def train_loop_per_worker():
                         )
                     elif isinstance(batch, np.ndarray):
                         bytes_read += batch.nbytes
+                    elif isinstance(batch, dict):
```

Comment:
It's only possible for this to be a dict of numpy arrays? Any other possible formats?

Reply:
With strict mode enabled, this should be defaulting to `numpy` as the default batch format. We missed a case with the `DataIterator` where we did not enforce this default batch format, which we are fixing in a separate PR: https://github.com/ray-project/ray/pull/36686

So, once the above PR is merged, the numpy format will no longer be an issue, as the default batch format will now be numpy.